<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1032</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MD TechView Concept - NOT FINAL SOFTWARE</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Recovery Guru</string>
       </attribute>
       <widget class="QWebView" name="webView">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>991</width>
          <height>661</height>
         </rect>
        </property>
        <property name="url">
         <url>
          <string>about:blank</string>
         </url>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Array Profile</string>
       </attribute>
       <widget class="QTextBrowser" name="textSAP">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>981</width>
          <height>711</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>MEL</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QsciScintilla" name="textMEL">
          <property name="font">
           <font>
            <family>Consolas</family>
            <pointsize>8</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>Parse into list</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>State Capture</string>
       </attribute>
       <widget class="QsciScintilla" name="textStateCap">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>991</width>
          <height>681</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>iSCSI Sessions</string>
       </attribute>
       <widget class="QsciScintilla" name="textiSCSI">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>991</width>
          <height>681</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <italic>false</italic>
          <bold>false</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="toolTip">
         <string/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelPath">
      <property name="text">
       <string>Current Bundle Path:  </string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Open Bundle</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1032</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWebView</class>
   <extends>QWidget</extends>
   <header>QtWebKit/QWebView</header>
  </customwidget>
  <customwidget>
   <class>QsciScintilla</class>
   <extends>QFrame</extends>
   <header>Qsci/qsciscintilla.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
